spec/system/custom/polls/polls_spec.rb

```diff
@@ -182,8 +185,43 @@
 
       click_button "Vote"
 
+      expect(page).to have_content "2 errors prevented your answers from being saved. Please check the " \
+                                   "marked fields to know how to correct them:"
+    end
+
+    scenario "Does not persist user answers when the terms and conditions are not accepted" do
+      poll = create(:poll)
+      create(:poll_question, :yes_no, poll: poll)
+      visit poll_path(poll)
+
+      choose "Yes"
+      check "By answering you accept the terms and conditions of use"
```

Comment:
I think this line was added accidentally :thinking:, since we're testing what happens when the conditions are **not** accepted.